### PR TITLE
First iteration of the landing page

### PR DESCRIPTION
**Pull request by gfontorbe**
This PR contains the first iteration for the landing page so that other contributors can review and comment on the content

### CODE REVIEW

File: hugo/content/_index.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        </div>
+        <p class="text-sprottyTeal text-xl text-center pb-20 font-medium">
+            Sprotty is an open-source project enabling the creation of powerful diagramming tools and graphical editors.
+            <br><br>
+            The project is maintained by <a href="https://www.typefox.io" class="underline decoration-dashed">TypeFox</a> and <a href="https://eclipsesource.com/" class="underline decoration-dashed">EclipseSource</a>, offering professional support to elevate your project to new heights!
+        </p>
+        <div class="flex flex-row gap-10 justify-center items-center pb-10">
+            <a href="/docs/getting_started" class="bg-sprottyMidBlue bg-opacity-50 rounded-xl px-8 py-3 text-sprottyDarkBlue shadow-md">Get started</a>
+            <div class="flex flex-row gap-2">
+                <a href="#what-is-sprotty" class="text-sprottyDarkBlue">Learn more</a>
```
Comment (spoenemann) [inline hugo/content/_index.html:24]: Nice to have: smooth scrolling like on langium.org

File: hugo/content/_index.html [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+            </p>
+        </div>
+    </div>
+</section>
+
+
+<section id="client">
+    <div class="flex flex-col">
+        <div class="py-8">
+            <p class="text-sprottyTeal text-center font-montserrat">The following example shows a full integration between Sprotty and the Monaco Editor. The Monaco Editor runs a language server for a DSL created with <a href="https://langium.org/">Langium</a> for creating state-machines. Edit the content of the editor and see changes transferred in real-time to the diagram!</p>
```
[the remark this answers is not in the document]
Comment (spoenemann) [reply hugo/content/_index.html:108]: The link to Langium does not match the link style used elsewhere on the page.

File: hugo/content/_index.html [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+<section id="introduction">
+    <div class="flex flex-col pt-10 px-20 pb-20 font-redHat">
+        <h2 class="text-sprottyTeal text-center font-medium text-3xl pb-10">
+            Diagram Vizualization Tools <br />for your web applications
+        </h2>
+        <div class="flex flex-row justify-center pb-5">
+            <img src="/assets/sprotty-multicore.png" alt="" class="mb-3 z-40" />
+            <img src="/assets/sprotty-xtext.png" alt="" class="mx-[-50px] mt-3 z-30" />
+            <img src="/assets/sprotty-dependency.png" alt="" class="mb-3 z-20" />
```
Comment (spoenemann) [inline hugo/content/_index.html:14]: Please use higher-resolution versions of these screenshots (see [README](https://github.com/eclipse-sprotty/sprotty#readme)).

File: hugo/content/_index.html [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+    </div>
+</section>
+
+<section id="characteristics">
+    <div class="flex flex-row gap-5 bg-im px-10 mb-[-3rem]">
+        <div class="basis-1/2">
+            <img src="/assets/sprotty-dependency.png" alt="" class="mt-[-4rem]"/>
+            <img src="/assets/sprotty-multicore.png" alt="" class="mt-[-5rem] pl-36 z-10 relative"/>
+            <img src="/assets/sprotty-yang.png" alt="" class="mt-[-5rem]"/>
+            <img src="/assets/sprotty-xtext.png" alt="" class="mt-[-5rem] pl-36"/>
```
Comment (spoenemann) [inline hugo/content/_index.html:54]: Please use higher-resolution versions of these screenshots (see [README](https://github.com/eclipse-sprotty/sprotty#readme)).

**Review by spoenemann — COMMENTED** [review]
Thank you @gfontorbe! The front page looks really good. Apart from the minor remarks below, we should optimize the page for small displays.

**Comment by jonah-iden** [comment]
i just had a quick glance, but what instantly bothers me alot is that the Documentation is vertically not in the middle
grafik

File: hugo/content/_index.html [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+<section id="outro">
+    <div class="flex flex-col bg-sprottyLightBlue bg-opacity-30 items-center text-sprottyDarkBlue text-center py-20 mt-[-3rem]">
+        <p class="font-montserrat text-xl pb-8 px-32">Sprotty was first released in 2017 and has since been continuously developed and improved by <a href="https://www.typefox.io">TypeFox</a>, <a href="https://eclipsesource.com/">EclipseSource</a>, and many individual contributors.</p>
+        <img src="/assets/sprotty-logo-fish.svg" alt="" class="w-[155px]">
+        <p class="font-redHat text-2xl py-8">Start using Sprotty right now or become a part of the sprotty school
+        </p>
+        <div class="flex flex-row gap-8">
+            <a href="https://www.npmjs.com/package/sprotty"><img src="assets/npm.svg" alt=""></a>
+            <a href=""><img src="assets/twitter.svg" alt=""></a>
```
[the remark this answers is not in the document]
Comment (gfontorbe) [reply hugo/content/_index.html:143]: I'll leave in here as a placeholder, might be replaced by links to TypeFox/EclipseSource?

**Pull request merged**